de.metas.vertical.pharma/src/main/java/de/metas/vertical/pharma/PharmaVendorPermissions.java

```diff
@@ -54,6 +54,9 @@ public static PharmaVendorPermissions of(final I_C_BPartner bpartner)
 		{
 			permissionsBuilder.add(PharmaVendorPermission.PHARMA_WHOLESALE);
 		}
+		if (bpartner.isPharmaVendorNarcoticsPermission()) {
```

Comment:
```suggestion
		if (bpartner.isPharmaVendorNarcoticsPermission()) 
		{
```
please take a look and see if you have the metasfresh (eclipse-) formatting rules from metasfresh-dev.
The current formatting rules are not written in stone, but everyone should use the same

Reply:
My bad, i forgot to indent there properly. I do have the formatting rules selected.